### PR TITLE
Add sberdataprocessor

### PR DESCRIPTION
**Pull request by amp1re**
add SberDataProccessors and tests

### CODE REVIEW

File: ml_package_framework/data_processing/base.py [hunk 1]
```diff
@@ -9,10 +9,11 @@ class DataProcessor(ABC):
     """
 
     @abstractmethod
-    def load_data(self, *args, **kwargs):
+    def load_data(self, path, *args, **kwargs):
```
Comment (ieBoytsov) [inline ml_package_framework/data_processing/base.py]: add typing here path: str

File: ml_package_framework/data_processing/data_processors.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+from .base import DataProcessor
+
+
+class SberDataProcessor(DataProcessor):
+    """
+    SberDataProcessor is a subclass of DataProcessor for handling data specifically from Sber.
+    """
+
+    def load_data(self, path: str, *args, **kwargs) -> pd.DataFrame:
```
Comment (ieBoytsov) [inline ml_package_framework/data_processing/data_processors.py:34]: typing path: str

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,71 @@
+alabaster==0.7.13
```
Comment (ieBoytsov) [inline requirements.txt]: hmm how did you get this list? you should store here only primary dependencies.

File: tests/test_data_processing/test_data_processors.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    # Setup
+    processor = SberDataProcessor()
+    csv_content = "col1,col2\nvalue1,value2\nvalue3,value4"
+    temp_csv_path = create_temp_csv(csv_content)
+
+    # Execute
+    data = processor.load_data(temp_csv_path)
+
+    # Verify
+    assert isinstance(data, pd.DataFrame)
```
Comment (ieBoytsov) [inline tests/test_data_processing/test_data_processors.py]: see pandas.testing assert_frames_equal

File: requirements.txt (hunk 3 above)
Comment (amp1re) [reply requirements.txt]: So I fix it with pipreqs --force

File: ml_package_framework/data_processing/base.py (hunk 1 above)
Comment (amp1re) [reply ml_package_framework/data_processing/base.py]: fixed it

File: ml_package_framework/data_processing/data_processors.py (hunk 2 above)
Comment (amp1re) [reply ml_package_framework/data_processing/data_processors.py:34]: here it is

File: requirements_dev.txt [hunk 5]
```diff
@@ -1,12 +0,0 @@
-pip==19.2.3
-bump2version==0.5.11
-wheel==0.33.6
-watchdog==0.9.0
-flake8==3.7.8
```
Comment (ieBoytsov) [inline requirements_dev.txt:5]: I would advise to keep fixed versions of linters and tests (flake8 and other). 

Delete here: Click, tox, twine,  watchdog, bump2version. All thee rest better leave as is.

**Pull request merged**